docs/src/api/class-locatorassertions.md

```diff
@@ -820,6 +820,58 @@ Expected class or RegExp or a list of those.
 ### option: LocatorAssertions.toHaveClass.timeout = %%-js-assertions-timeout-%%
 ### option: LocatorAssertions.toHaveClass.timeout = %%-csharp-java-python-assertions-timeout-%%
 
+## async method: LocatorAssertions.toContainClass
+* langs:
+  - alias-java: containClass
+
+Ensures the [Locator] points to an element with classList which contains given CSS class.
+In contrast to toHaveClass, toContainClass checks just one class on element with many classes.
+
+```js
+const locator = page.locator('#component');
+await expect(locator).toContainClass('selected');
+```
+
+```java
+assertThat(page.locator("#component")).hasClass(Pattern.compile("selected"));
+```
+
+```python async
+from playwright.async_api import expect
+
+locator = page.locator("#component")
+await expect(locator).to_have_class(re.compile(r"selected"))
```

Comment:
make sure to also use the right wording for the examples.

Reply:
Yeah) Im not sure I unterstand how that magic works, but probably toContainClass should be reflected as to_contain_class for python and ToContainClassAsync for csharp.

Reply:
yup!

Reply:
fixed